src/main/java/org/gridsuite/shortcircuit/server/repositories/ShortCircuitAnalysisResultRepository.java

```diff
@@ -248,52 +254,156 @@ public Optional<ShortCircuitAnalysisResultEntity> findResultsWithLimitViolations
     }
 
     @Transactional(readOnly = true)
-    public Page<FaultResultEntity> findFaultResultsPage(ShortCircuitAnalysisResultEntity result, List<ResourceFilter> resourceFilters, Pageable pageable, FaultResultsMode mode) {
+    public Page<FaultResultEntity> findFaultResultsPage(ShortCircuitAnalysisResultEntity result,
+                                                        List<ResourceFilter> resourceFilters,
+                                                        Pageable pageable,
+                                                        FaultResultsMode mode) {
         Objects.requireNonNull(result);
-        Specification<FaultResultEntity> specification = FaultResultSpecificationBuilder.buildSpecification(result.getResultUuid(), resourceFilters);
+
+        Optional<Sort.Order> childrenSort = extractChildrenSort(pageable);
+
+        Pageable modifiedPageable = addDefaultSort(filterOutChildrenSort(pageable, childrenSort),
+                DEFAULT_FAULT_RESULT_SORT_COLUMN);
```

Comment:
The default sort was there before, I didn't add it. I guess the point is to always have at least one default sort just in case. But it has nothing to do with children sort. The children sort is specifically filtered out.

Reply:
I forgot why I said that, I think I read too fast missunderstood, nvm